bindings/wasm/src/did/wasm_iota_did.rs

```diff
@@ -94,14 +94,14 @@ impl From<IotaDID> for WasmDID {
 /// Duck-typed union to pass either a string or WasmDID as a parameter.
 #[wasm_bindgen]
 extern "C" {
-  #[wasm_bindgen(typescript_type = "DID | string")]
-  pub type UWasmDID;
+  #[wasm_bindgen(typescript_type = "CoreDID | string")]
+  pub type UWasmIotaDID;
```

Comment:
```suggestion
  pub type UWasmCoreDID;
```

Shouldn't it be that?

Reply:
I think it should actually change to `typescript_type = "IotaDID | string"`, since `UWasmIotaDID` is still used in places that expect `IotaDID` not `CoreDID`.